app/admin/dashboard.rb

```diff
@@ -42,8 +42,8 @@ def set_dashboard_values
       }
 
       @practices_favorited_stats = {
-        favorited_this_month: UserPractice.where(created_at: @beginning_of_current_month..@end_of_current_month, favorited: true).count,
-        favorited_one_month_ago: UserPractice.where(created_at: @beginning_of_last_month..@end_of_last_month,favorited: true).count,
+        favorited_this_month: UserPractice.where(time_favorited: @beginning_of_current_month..@end_of_current_month).count,
```

Comment:
Question... (sorry if I missed this in a mtg or something... 😅) but why do we want to count in the favorited count (for the current month and last month) practices that are no longer favorited by a user?? 🤔

Reply:
bc we still want to capture that the practice was favorited in the month columns... but we do decrement the favorite total_count when the practice is UN-favorited.

Reply:
ahh ok! thanks, brad! :)